com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef

```diff
@@ -17,6 +17,12 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.recorder",
+            "expression": "2.2.0-preview.4",
```

Comment:
What's the interpretation of 'expression' -- is it a minimum version, or an exact match?

Reply:
it's minimum version, you can go higher

Reply:
for reference (under Version Defines section): https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html